### ISSUE TITLE
add particle species in `ParticleBeam`

### ISSUE THREAD

**Issue by jp-ga** [desy-ml/cheetah#231]
I have noticed that `ParticleBeam` doesn't have a particle mass property. It ends up being necessary to assume particles are electrons and then import the electron mass for different elements/tracking routines. It would be useful to add the particle mass or species to `ParticleBeam` and use electrons as default.

**Comment by jp-ga** [comment]
Any thoughts on this @jank324 @cr-xu ?

**Comment by jank324** [comment]
> Any thoughts on this @jank324 @cr-xu ?

This was discussed somewhere before, I don't remember where. In the end we didn't do it (yet) because we didn't need this yet.

I don't see any reason why we shouldn't add it.

**Comment by cr-xu** [comment]
Yes, we should add this. It's just been on the to-do list for a while.
